applications/convection_diffusion_application/python_scripts/check_and_prepare_model_process_convection_diffusion.py

```diff
@@ -31,13 +31,16 @@ def __init__(self, main_model_part, Parameters):
 
     def Execute(self):
         
-        problem_domain_sub_model_parts = []
-        for i in range(self.problem_domain_sub_model_part_list.size()):
-            problem_domain_sub_model_parts.append(self.main_model_part.GetSubModelPart(self.problem_domain_sub_model_part_list[i].GetString()))
+        #problem_domain_sub_model_parts = []
```

Comment:
Why commenting?

Reply:
i believe that we shoild actjally using the main model part directly, nothing special is needed in this case. i am commenting since i am not 100% sure what i did always works